tasks/src/robocorp/tasks/_task_options.py

```diff
@@ -0,0 +1,6 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class TaskOptions:
```

Comment:
Actually, I was thinking, this isn't correct at this level. 

For the `task`, `is_consequential` does not make sense, just in the `actions`.

Thus, in the tasks we just want the options to be stored as a regular dict with the options the user entered (no typing or checks done as to what the options are).

Reply:
p.s.: at the `task` level, the feature is that `options` are now available to be passed in the decorator (and clients can then introspect those at `task.options` if they want to handle them) -- please add an entry to the tasks/docs/changelog.md outlining this change.

Reply:
Should be done

1. removed `TaskOptions`
2. args passed to `@task` will be accessible through `task.options`
3. added `ActionOptions` dataclass